deploy/charts/rook-ceph/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if and .Values.monitoring.enabled .Values.csi.serviceMonitor.enabled (or .Values.csi.enableLiveness .Values.csi.enableGrpcMetrics) }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: csi-metrics
+  namespace: {{ .Release.Namespace }}
+  labels: {{- include "library.rook-ceph.labels" . | nindent 4 }}
+spec:
+  namespaceSelector:
+    matchNames:
+      - {{ .Release.Namespace }}
+  selector:
+    matchLabels:
+      app: csi-metrics
+  endpoints:
+    {{- if .Values.csi.enableLiveness }}
+    - port: csi-http-metrics
```

Comment:
This metrics port is only available if the liveness sidecar is enabled?

Reply:
conditionnal in the deployment : https://github.com/rook/rook/blob/dc275f3513819f7c878ed7f850f85800a15983d9/pkg/operator/ceph/csi/template/cephfs/csi-cephfsplugin.yaml#L122

but both are always enabled in the service :
https://github.com/rook/rook/blob/dc275f3513819f7c878ed7f850f85800a15983d9/pkg/operator/ceph/csi/template/cephfs/csi-cephfsplugin-svc.yaml , hence my original commit without conditionnals

Reply:
Ok, nice to have the conditionals since helm is more flexible.